From 687e4b0b6459228a2c64b6a84b09c534c502942a Mon Sep 17 00:00:00 2001
From: arter97 <qkrwngud825@gmail.com>
Date: Sat, 27 Aug 2016 10:08:53 -0400
Subject: [PATCH] Create 0 compression ratio jar files

Modern devices now has a blazing fast NAND that storage speed is not the bottle-neck when it comes to reading apks.

So use 0 compression ratio for improving speed and memory usage.

Change-Id: Ie5f0357145603c80042be77299d169c5ac7c1d89
Signed-off-by: arter97 <qkrwngud825@gmail.com>
Signed-off-by: Chet Kener <Cl3Kener@gmail.com>

Conflicts:
	core/definitions.mk
Signed-off-by: sub77 <sub77@ymail.com>
---
 core/definitions.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/definitions.mk b/core/definitions.mk
index 84ea80197..b58f202f2 100644
--- a/core/definitions.mk
+++ b/core/definitions.mk
@@ -2423,7 +2423,7 @@ endef
 
 # $(1): the package file.
 define add-dex-to-package-arg
-$(hide) find $(dir $(PRIVATE_DEX_FILE)) -maxdepth 1 -name "classes*.dex" | sort | xargs zip -qjX $(1)
+$(hide) zip -0qj $@ $(dir $(PRIVATE_DEX_FILE))classes*.dex
 endef
 
 # Add java resources added by the current module.
-- 
2.13.0

